qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -382,6 +382,10 @@ class BaseSpecification extends Specification {
         orchestrator.deleteSecret("gcr-image-pull-secret", Constants.ORCHESTRATOR_NAMESPACE)
     }
 
+    static Boolean isPostgresRun() {
```

Comment:
❤️ 
Could you replace all `Env.CI_JOBNAME.contains("postgres")` with this method.

Reply:
Could be in another PR

Reply:
Will do in a separate PR.